tests/unit-core/test-api-set-and-clear-error-flag.c

```diff
@@ -27,10 +27,12 @@ main (void)
   jerry_value_set_error_flag (&obj_val);
   jerry_value_t err_val = jerry_acquire_value (obj_val);
 
-  jerry_value_clear_error_flag (&obj_val);
-  jerry_release_value (obj_val);
+  obj_val = jerry_get_value_from_error (obj_val, true);
```

Comment:
I think the obj_val should be err_val in `jerry_get_value_from_error (obj_val, true);`,